File: C++/LinkedList/Single/Runner.cpp
#include <iostream>
#include "SinglyLinkedList.h"

int main() {
   SinglyLinkedList ll;
   ll.insertElement(5, 0);
   ll.insertElement(10, 1);
   ll.insertElement(15, 2);
   ll.insertElement(20, 3);

   ll.traverse();

   ll.deleteElement(0);
   ll.traverse();
   return 0;
}

File: Java/src/main/java/com/DSA/Java/LinkedList/Double/Circular/Runner.java
package com.DSA.Java.LinkedList.Double.Circular;

public class Runner {
   public static void main(String[] args) {
      CircularDoublyLinkedList cDL = new CircularDoublyLinkedList();
      cDL.createCircularDoublyList(10);
      cDL.insertElement(5, 0);
      cDL.insertElement(15, 2);
      cDL.insertElement(20, 3);
      cDL.insertElement(25, 4);
      
      cDL.traverse();
      // cDL.deleteList();
      // cDL.traverse();

      // cDL.deleteElement(1);
      // cDL.deleteElement(1);
      // cDL.traverse();
      // cDL.searchElement(20);
      // cDL.searchElement(21);
   }
}
